Changelog.md

```diff
@@ -1,3 +1,11 @@
+Unreleased
+---------------------------
+
+* [BACKWARD INCOMPATIBLE] Encode/decode GCP stack names
```

Comment:
Might perhaps want to be a bit more specific here: if I understand correctly, the compatibility breaking change _only_ affects GCP labs, whereas OpenStack labs should be unaffected. Correct?

Reply:
Correct, it only affects GCP labs, and since there are no known runs of it in production, 3.0.0 was only just released, and there's no non-goldbergian way to make this backward compatible, I figured "what the heck".

Reply:
But yes, this deserves clarification.  Ammending.